index.js

```diff
@@ -239,4 +239,20 @@ app
   .use(router.routes())
   .use(router.allowedMethods());
 
-app.listen(process.env.BIND_PORT || "3000", process.env.BIND_ADDRESS || "0.0.0.0");
+let bindAddress = process.env.BIND_ADDRESS || "0.0.0.0";
```

Comment:
this can be a "const" no? it's defined at the startup and the variable does not change.